gradle-plugin/plugin/src/main/kotlin/com/emergetools/android/gradle/util/Git.kt

```diff
@@ -11,7 +11,10 @@ internal object Git {
   }
 
   fun baseSha(): String? {
-    return "git merge-base ${remoteHeadBranch()} ${currentBranch()}".execute().trimmedText
+    val baseSha = "git merge-base ${remoteHeadBranch()} ${currentBranch()}".execute().trimmedText
```

Comment:
Noticed this was setting the sha to an empty string if no output was generated, so added a simple check to explicitly return null.